ansible_collections/arista/avd/roles/eos_designs/templates/network_services/route-maps.j2

```diff
@@ -1,19 +1,19 @@
 {# Route-maps for tenant bgp peers set_ipv4_next_hop parameter #}
 route_maps:
-{% for tenant in switch.tenants | arista.avd.natural_sort %}
-{%     for vrf in switch.tenants[tenant].vrfs | arista.avd.natural_sort %}
-{%         for bgp_peer in tenants[tenant].vrfs[vrf].bgp_peers | arista.avd.natural_sort %}
-{%             if inventory_hostname in tenants[tenant].vrfs[vrf].bgp_peers[bgp_peer].nodes | arista.avd.natural_sort %}
-{%                 if tenants[tenant].vrfs[vrf].bgp_peers[bgp_peer].set_ipv4_next_hop is arista.avd.defined or tenants[tenant].vrfs[vrf].bgp_peers[bgp_peer].set_ipv6_next_hop is arista.avd.defined %}
-  RM-{{ vrf }}-{{ bgp_peer }}-SET-NEXT-HOP-OUT:
+{% for tenant in network_services_data.tenants %}
```

Comment:
Here it is better to sort than in the logic.j2

Reply:
you would have to add the sort on every level, so in this example on tenants and on vrfs. Many of the others also use SVIs and l2vlans, so it becomes more expensive.